src/components/RadioButton/RadioButton.tsx

```diff
@@ -1,13 +1,14 @@
-import React, { forwardRef, useCallback, useMemo, useRef } from "react";
 import cx from "classnames";
+import React, { forwardRef, useCallback, useMemo, useRef } from "react";
 import useMergeRefs from "../../hooks/useMergeRefs";
 import Clickable from "../Clickable/Clickable";
 import { backwardCompatibilityForProperties } from "../../helpers/backwardCompatibilityForProperties";
 import { baseClassName } from "./RadioButtonConstants";
```

Comment:
Btw if instead of this you create in this file `const CSS_BASE_CLASS = "monday-style-radio-button-component";`, or `const *prefix*_CSS_BASE_CLASS = "monday-style-radio-button-component";`,  then the script will replace it all the templateLiterals (\`${}...\`) with just a string with full class name, which might be more convenient